web/views.py

```
@@ -3685,3 +3665,161 @@ def donation_success(request):
 def donation_cancel(request):
     """Handle donation cancellation."""
     return redirect("donate")
+
+
+@login_required
+def create_group_enrollment(request, course_id):
+    course = get_object_or_404(Course, id=course_id)
+    discount = Discount.objects.get(discount_type="group", course=course)
+    
+    context = {
```

Ruff (0.8.2)

Local variable `context` is assigned to but never used

Remove assignment to unused variable `context`

(F841)